Form/TranslatableType.php

```diff
@@ -12,7 +12,9 @@
 namespace Nfq\AdminBundle\Form;
 
 use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
 use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
 use Symfony\Component\OptionsResolver\OptionsResolverInterface;
```

Comment:
`use Symfony\Component\OptionsResolver\OptionsResolverInterface` is not used anymore?